stack.py

```diff
@@ -57,22 +57,35 @@ def parse_program(code):
         parts = line.strip().split()
         if not parts:
             continue
-        op = parts[0]
-        args = [float(arg) for arg in parts[1:]]
-        yield Instr(op, args)
+        # TODO: This only checks for 'quiet' as a prefix but should allow
+        # "#" and "♯" as equlivlent (move it to Instr.)
+        if parts[0] == 'quiet':
+            prefix = ['quiet']
+            op = parts[1]
+            args = [try_float(arg) for arg in parts[2:]]
+        else:
+            prefix = None
```

Comment:
The default value for prefix should be `[]`, since it's a collection of nothing, not no collection.